scripts/packageAdBlock.js

```diff
@@ -34,16 +35,20 @@ async function stageFiles (version, outputDir) {
 }
 
 const postNextVersionWork = (componentSubdir, key, publisherProofKey,
-  binary, localRun, version) => {
+  binary, localRun, version, contentHash) => {
   const stagingDir = path.join('build', 'ad-block-updater', componentSubdir)
   const crxOutputDir = path.join('build', 'ad-block-updater')
   const crxFile = path.join(crxOutputDir, `ad-block-updater-${componentSubdir}.crx`)
+  const contentHashFile = path.join(crxOutputDir, `ad-block-updater-${componentSubdir}.contentHash`)
```

Comment:
I'm not sure `contentHash` could be `undefined`
but how about trying to delete `contentHashFile` at the beginning for safe?
If there is existing file and `contentHash` is `undefined`, it could cause unexpected behavior.

Reply:
Good suggestion, I've added an additional check to remove the file if it already exists